Tasks/PackerBuild/src/taskParameters.ts

```diff
@@ -51,21 +51,35 @@ export default class TaskParameters {
                     this._extractImageDetails();
                 }              
 
-                this.packagePath = this._getResolvedPath(tl.getVariable('System.DefaultWorkingDirectory'), tl.getInput(constants.DeployPackageInputName, true));
-                console.log(tl.loc("ResolvedDeployPackgePath", this.packagePath));
-                var deployScriptAbsolutePath = this._getResolvedPath(this.packagePath, tl.getInput(constants.DeployScriptPathInputName, true));
-                var scriptRelativePath = path.relative(this.packagePath, deployScriptAbsolutePath);
-                this.deployScriptPath = this._normalizeRelativePathForTargetOS(scriptRelativePath);
-                console.log(tl.loc("ResolvedDeployScriptPath", this.deployScriptPath));
+                try {
```

Comment:
instead of wrapping each action with a try catch for error context, why not just console.log('Resolving Deploy Package') etc...  That way you just have one outer try catch and if any step fails, you have error context because it will be just after the contextual message.

So instead of wrapping every action ...

```
try {
     console.log('Doing xxx');
     doX();
     console.log('Doing yyy');
     doY();
} catch {
      console.error(error);
      tl.setResult(failed,....);
}
```